example/contracts/src/main/kotlin/io/p8e/contracts/example/HelloWorld.kt

```diff
@@ -5,10 +5,12 @@ import io.p8e.annotations.Function
 import io.p8e.annotations.Input
 import io.p8e.annotations.Participants
 import io.p8e.annotations.ScopeSpecification
+import io.p8e.annotations.ScopeSpecificationDefinition
 import io.p8e.proto.ContractScope.Scope
 import io.p8e.proto.ContractSpecs.PartyType.*
 import io.p8e.proto.example.HelloWorldExample.ExampleName
 import io.p8e.spec.P8eContract
+import io.p8e.spec.P8eScopeSpecification
 
 @Participants(roles = [OWNER])
 @ScopeSpecification(names = ["io.p8e.contracts.example.helloWorld"])
```

Comment:
would it be possible to change this so we just reference the @ScopeSpecificationDefinition annotated class (i.e. HelloWorldScopeSpecification) here instead of having to duplicate the name string over? Then we could use that class reference to grab the annotation details and fetch the name off of that (and maybe event make the name optional and just use the class' fully-qualified name instead (i.e. io.p8e.contracts.example.HelloWorldScopeSpecification in this case)?